bakery/src/pipelines/distribution.js

```diff
@@ -25,24 +28,49 @@ const pipeline = (env) => {
       }
     },
     {
-      name: 's3-feed',
+      name: 's3-queue',
       type: 's3',
       source: {
-        bucket: env.S3_DIST_BUCKET,
-        versioned_file: env.VERSIONED_FILE,
+        bucket: env.S3_VERSIONED_BUCKET,
+        versioned_file: versionedFile,
+        initial_version: 'initializing',
         access_key_id: awsAccessKeyId,
         secret_access_key: awsSecretAccessKey
       }
+    },
+    {
+      name: 'ticker',
+      type: 'time',
+      source: {
+        interval: env.PIPELINE_TICK_INTERVAL
+      }
     }
   ]
 
+  const feederJob = {
+    name: 'feeder',
+    plan: [
+      { get: 'ticker', trigger: true },
```

Comment:
We will miss books in the queue if we don't use `version: every` here
```suggestion
      { get: 'ticker', trigger: true, version: 'every' },
``

Reply:
This is for the timer tick versus the file, though. I think the only reason I didn't include the "every" here was in case it was delayed and there were two "ticks" too close (maybe this isn't a real issue with the time resource, though?)

The file resource version does have "every" on line 70 below.

Reply:
Oops! Wrong resource! There is `version: every` on the resource that actually matters!